Add Deploy 13.3.0 and 14.2.0 release notes and document ImportOnStartup and ValidateDependenciesOnImport settings (#6699)
umbraco/UmbracoDocs: 15/umbraco-deploy/deployment-workflow/import-export.md

If your account has access to the Settings section, you can also choose to include the schema information and related files as well.

![Tree export](images/tree-export-modal.png)

> When doing an environment export, you also have the option to include *all* schema in the export, regardless of whether it is in use by any content.

Reviewer: ```suggestion
When exporting an environment, you can include _all_ schema, regardless of whether any content uses it.
```
